### ISSUE TITLE
Open Logic App by ID

### ISSUE THREAD

**Issue by studiology** [microsoft/vscode-azurelogicapps#246]
The Open Logic Apps in Designer / Editor operations only allow selections from the list displayed in the tree view.

I have *loads* of logic apps in my subscription, and would prefer not to have to click "Load More..." 50+ times before the logic app I'm after appears in the list.

Typing the logic app name into the selector only filters from that existing list. It would be ideal if we could search by name or enter the full ID of the logic app (ie. `/subscriptions/{subscriptionid}/resourceGroups/{rg}/providers/Microsoft.Logic/workflows/{workflowName}`).

Otherwise filtering by resource group as well as subscription in the tree view pane would work - as would using the resource groups extension's tree view pane to open a Logic App - or tell the logic apps extension of its existence so it appears in the logic apps pane.

Optionally caching the list of logic apps might be workable also, or an option to have the `Load More...` button retrieve a configurable number of pages of data

**Comment by joechung-msft** [comment]
> Filtering by resource group as well as subscription in the tree view pane would work - as would using the resource groups extension's tree view pane to open a Logic App - or tell the logic apps extension of its existence so it appears in the logic apps pane.

https://github.com/microsoft/vscode-azureresourcegroups/issues/135 sounds like the way to go.

**Comment by studiology** [comment]
> > Filtering by resource group as well as subscription in the tree view pane would work - as would using the resource groups extension's tree view pane to open a Logic App - or tell the logic apps extension of its existence so it appears in the logic apps pane.
> 
> [microsoft/vscode-azureresourcegroups#135](https://github.com/microsoft/vscode-azureresourcegroups/issues/135) sounds like the way to go.

Yes - though it'd be good to also have an "open Logic App by ID" operation
